### DISCUSSION TITLE
How to use this request

### DISCUSSION THREAD

**Discussion by bewareilive** [mockoon/mockoon#812] [Help] [answered]
api1. post
      127.0.0.1/api/v3:person
api2. post
      127.0.0.1/api/v3:car

my  mockoon: 
![image](https://user-images.githubusercontent.com/26020170/188396314-e240716f-82f5-4d18-a8a5-7464ba9ac6a1.png)

Whether the first or the second is requested, the first is response

**Comment by 255kb** [comment, marked as answer]
I think your issue is caused by colons being considered as path params.
Trying enclosing the colon in brackets, like this: `v3[:]person`

**Reply by bewareilive** [reply]
Thanks. The request is successful.Is [] an escape character?

**Reply by 255kb** [reply]
Yes, you can read this in the [documentation](https://mockoon.com/docs/latest/routing/). Mockoon is based on Expressjs.
